api/database_cluster_restore.go

```diff
@@ -71,7 +71,7 @@ func (e *EverestServer) CreateDatabaseClusterRestore(ctx echo.Context) error {
 	if dbCluster.Status.Status == everestv1alpha1.AppStateRestoring {
 		e.l.Error("failed creating restore because another one is in progress")
 		return ctx.JSON(http.StatusBadRequest, Error{
-			Message: pointer.ToString("Another restore is in progress"),
+			Message: pointer.ToString("Another restore process for this DB cluster is currently in progress. Wait for its completion before initiating another."),
```

Comment:
```suggestion
			Message: pointer.ToString("The selected database cluster is being restored. Please wait until the restoration process is finished."),
```

Both the old and the new error message were fine I think. I suggested one more.
In general I would advise for the end-user error message to be handled by the frontend and we return a recognisable error code instead.

Reply:
Hi Michal, thank you for your suggestion. I also like your suggested message. However, this message was written by Peter and Catalina and I don't want to overrule their decision. Since she's on vacation, I'll keep the current string for now.

Regarding the FE handling the error messages I also agree with that but I think there might be situations where we might need some extra context from the BE, it depends on the situation.